src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import IconSvg from '../IconSvg'
+import MarkdownSynapse from './MarkdownSynapse'
+
+export enum MarkdownEditorTabs {
+  WRITE = 'WRITE',
+  PREVIEW = 'PREVIEW',
+}
+
+export type MarkdownEditorProps = {
+  placeholder?: string
+}
+
+export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
+  placeholder,
+}) => {
+  const [currentTab, setCurrentTab] = useState<MarkdownEditorTabs>(
+    MarkdownEditorTabs.WRITE,
+  )
+  const [text, setText] = useState<string>()
+
+  return (
+    <div className="bootstrap-4-backport MarkdownEditor">
+      <div className="TopController">
```

Comment:
`"TopController"` is confusing, since controller is an overloaded term. How about `"MarkdownEditorControls"`?